package.json

```diff
@@ -18,20 +18,20 @@
     "webpack-server": "grunt server",
     "webpack-build": "npm run webpack-browser && npm run webpack-module && npm run webpack-server",
     "lint": "balena-lint -e js -e ts src build typings Gruntfile.ts && npx tsc --project tsconfig.dev.json --noEmit",
-    "test": "npm run lint && npm run build && npm run webpack-build && npm run test:compose",
-    "test:compose": "trap 'docker-compose -f docker-compose.npm-test.yml down ; echo Stopped ; exit 0' SIGINT; docker-compose -f docker-compose.npm-test.yml up -d && sleep 2 && DATABASE_URL=postgres://docker:docker@localhost:5431/postgres npm run mocha",
+    "test": "npm run lint & npm run build && npm run webpack-build && npm run test:compose",
+    "test:compose": "trap 'docker-compose -f docker-compose.npm-test.yml down ; echo Stopped ; exit 0' SIGINT; docker-compose -f docker-compose.npm-test.yml up -d && sleep 2 && DATABASE_URL=postgres://docker:docker@localhost:5431/postgres PINEJS_WEBRESOURCE_MAXFILESIZE=1000000000 S3_ENDPOINT=http://localhost:43680 S3_ACCESS_KEY=USERNAME S3_SECRET_KEY=PASSWORD npm run mocha",
```

Comment:
Is there a value in testing for 1GiB rather than couple of MiBs?

Reply:
We need a way to validate we can upload large files (500MiB for example). But if we want to remove the test for uploading large files (which tbh I don't think we should), then we can lower this down.

Reply:
Yeah I agree we should keep the large file tests.